chainerui/models/result.py

```diff
@@ -43,17 +44,60 @@ def __init__(self, path_name=None, name=None, project_id=None,
     def __repr__(self):
         return '<Result id: %r, path_name: %r />' % (self.id, self.path_name)
 
-    @property
-    def serialize(self):
-        """serialize."""
+    @classmethod
+    def create(cls, path_name=None, name=None, project_id=None,
+               log_modified_at=None):
+        """Initialize an instance and save it to db."""
+        result = cls(path_name, name, project_id, log_modified_at)
+
+        DB_SESSION.add(result)
+        DB_SESSION.commit()
+
+        from chainerui.tasks import crawl_result
+        crawl_result(result.id, True)
```

Comment:
Calling `crawl_result` is necessary on here? I think that `crawl_result` is called after registering project, like here

https://github.com/gky360/chainerui/blob/9c326e393874cb1c2d1cad1750e72bd4e5365244/chainerui/views/result.py#L38

Reply:
I think we need to call `crawl_result` immediately after a result is created every time. So I want `crawl_result` to remain here.

Reply:
I ssee.